### ISSUE TITLE
Printable A-File

### ISSUE THREAD

**Issue by RocketMan** [RocketMan/zookeeper#42]
I have some doubts about the Print link (former Printable A-File) on the Adds page.

First, maybe we should just create a print stylesheet, such that doing a browser print action from the Current File page styles the output correctly, so that no visual element is necessary, and we can just get rid of it.

If we don't do that, then with the link now on the Adds page, it seems somewhat confusing, since it is not printing the Add on that page, but the whole file.  Also, it took me a while to find it.  I would be curious to see if the guy who prints out the a-file can find it without coaching.

**Comment by eric-gilbertson** [comment]
Agree that use of CSS for printing is the best solution. For now changing link to 'Print View' so that it matches the wording in the view playlist page.

**Comment by RocketMan** [comment]
I just committed a change that moves 'Print View' to the tab area (by RSS) on the Current File page, and conditions its display only if the logged in user has Add Manager privs.

I think this makes sense, as it is up out of the way of the page content, yet is on the page that is actually being printed (Current File versus Adds).  I also restored its limited visibility to the Add Manager people only.  If you think it would add value for everyone to see the link, let's do that.
